src/plans/D3MAaveV2Plan.sol

```diff
@@ -53,7 +53,7 @@ interface InterestRateStrategyLike {
     function getMaxVariableBorrowRate() external view returns (uint256);
 }
 
-contract D3MAavePlan is ID3MPlan {
+contract D3MAaveV2Plan is ID3MPlan {
```

Comment:
Thinking that this is already going to production, we might want to keep the original name. That will also reduce the amount of file changes in this PR.

Reply:
Won't that be more confusing to future users?

Reply:
Let's discuss it in chat.